libs/pages/dashboard/src/lib/dashboard-order-list/dashboard-order-list.tsx

```diff
@@ -0,0 +1,72 @@
+import {
+  Price,
+  StackedList,
+  UserAvatar,
+  UserName,
+  Users,
+} from '@pasnik/components';
+import { NavLink } from 'react-router-dom';
+import { OrderStatusBadge, OrderTimestamp } from '@pasnik/features/orders';
+import { CashIcon, OfficeBuildingIcon } from '@heroicons/react/outline';
+import { OrderModel } from '@pasnik/api/data-transfer';
+import { WorkspaceName } from '@pasnik/features/workspaces';
+
+export interface DashboardOrderListProps {
+  orders: OrderModel[];
+}
+
+export function DashboardOrderList({ orders }: DashboardOrderListProps) {
+  return (
+    <StackedList>
+      {orders.map((order) => (
+        <StackedList.Item
+          key={order.id}
+          wrapper={({ children }) => (
```

Comment:
useMemo

Reply:
It is inside loop function, could you provide an example